tests/_test_func_asyncio.py

```diff
@@ -51,10 +51,33 @@ def aioredis_pool():
         pytest.skip()
 
 
+@pytest.fixture()
+def aioredis_connection():
+    if sys.version_info <= (3, 5):
+        pytest.skip()
+
+    import aioredis
+
+    connection_coroutine = aioredis.create_redis(
+        ('localhost', 6379)
+    )
+
+    return connection_coroutine, ring.aioredis
+
+
+@pytest.fixture(params=[
+    lazy_fixture('aioredis_connection'),
+    lazy_fixture('aioredis_pool'),
+])
+def aioredis_clients(request):
```

Comment:
When resolving fixture, the fixture is always a client, not a sequence of clients. So `aioredis_client` would be better name in point of view of each test functions.